### ISSUE TITLE
Use of uninitialized variable in component not caught

### ISSUE THREAD

**Issue by billyjanitsch** [yannickcr/eslint-plugin-react#190]
```jsx
import React from 'react'
import Section from './section'

class MyComponent extends React.Component {
  render() {
    return (
      <Section header={myVar}>
        test
      </Section>
    )
  }
}

export default MyComponent
```

eslint doesn't catch the uninitialized `myVar`, even when `no-use-before-define` is enabled.

**Comment by yannickcr** [comment]
Hi,

The eslint rule to catch uninitialized variables is [no-undef](http://eslint.org/docs/rules/no-undef).

**Comment by tleunen** [comment]
Do you know if this is something that has been changed in eslint recently?
I never explicitly set this rule and everything worked fine but now I have to explicitly set the rule to have the warnings.

Might be because of a collision with this plugin?

**Comment by yannickcr** [comment]
Before `v1.0.0` eslint had a default configuration with some rules enabled.

Now all rules comes disabled by default, see the [eslint migration guide to 1.0.0](http://eslint.org/docs/user-guide/migrating-to-1.0.0).

**Comment by tleunen** [comment]
Ah ;) Thanks!
